src/core/utils/generateForgotPasswordMail.ts

```diff
@@ -0,0 +1,48 @@
+const GenerateForgotPasswordMail = (link: string): string => {
```

Comment:
I prefer html or ejs file

Reply:
พอจะมีแหล่ง source code มั้ยครับ เพราะมันเป็นส่งเมล ผมเลยไม่รู้จะให้มัน render ยังไง

Reply:
- https://notiz.dev/blog/send-emails-with-nestjs#handlebars-mail-template
- https://dev.to/brngranado/create-a-email-sender-with-nestjs-and-nodemailer-3jk1
- https://github.com/famesensor/Playgroud-NestJs-Document/blob/develop/src/transaction/transaction.service.ts